### ISSUE TITLE
lpadmin: separate enable and encrypt option?

### ISSUE THREAD

**Issue by senatormailman** [apple/cups#5308]
I can't see any previous issues regarding this, but this seems to have been the case for a bit. I understand that doing 'lpadmin -p $Printername -E' will enable the printer queue that had just been created while 'lpadmin -E -p $Printername' will just create the printer queue but the connection will be encrypted.

Is there a reason for this? Would separating the encrypt and enable options (to something like -E for enable and -e for encrypt) be breaking any kind of functionality?

Thanks for all the work!

**Comment by michaelrsweet** [comment]
@senatormailman This is a historical issue dating back to CUPS 1.1.x when we added support for encryption - lpadmin already supported a "-E" option so we opted to overload -E (-E before -p = use encryption, -E after -p = enable-and-accept) rather than use a different "use encryption" option for lpadmin vs. every other command (where -E means use encryption only).

"-e" actually is already used for System V lpadmin (-e copies the named printer to the new one) so we couldn't use that.
